.drone.yml

```diff
@@ -49,5 +49,6 @@ matrix:
     - { TARGET: test, BROWSER: firefox, CONSTEL: ui.grid }
     - { TARGET: test, BROWSER: firefox, CONSTEL: ui.scheduler }
     - { TARGET: test, BROWSER: firefox, CONSTEL: viz }
-    - { TARGET: test_functional }
+    - { TARGET: test_functional, CONSTEL: dataGrid }
```

Comment:
I suggest `COMPONENT` instead of `CONSTEL`.

We used `CONSTEL` ("constellation") to split unit tests because `category` and `suite` were already taken. Your parameter is `componentFolder` so I don't see a reason to use a fancy word.